packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -66,12 +71,13 @@ export const GroupTitleUI = styled( Text )`
 export const ItemUI = styled.li`
 	border-radius: 2px;
 	color: ${ G2.lightGray.ui };
+	margin-bottom: 0;
 
 	button,
 	a {
+		height: 32px;
```

Comment:
@jameskoster to confirm, but I remember agreeing to stick with the styles of the base components v. the design when they differ. That would be button's `components-button` class using `$button-size`, which I think is 36px.

Reply:
32px is good. Should we use min-height though, so that wrapping can occur for menu items with long labels?

Reply:
Good call @jameskoster . I've just committed this change :+1: